tests/init_test.go

```diff
@@ -93,7 +93,7 @@ type testMatcher struct {
 	failpat      []testFailure
 	skiploadpat  []*regexp.Regexp
 	slowpat      []*regexp.Regexp
-	whitelistpat *regexp.Regexp
+	allowlistpat *regexp.Regexp
```

Comment:
Here, too, the term 'allowlist' doesn't capture the meaning in the same way that 'whitelist' did. Since we are not using this feature at all, we might as well just delete it completely.